apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/dgs/DGSServiceTest.java

```diff
@@ -1200,12 +1218,14 @@ void testPurchaseWhenPriceQuantityExceedGoodsQuantity() {
 
     @Test
     void testDeliver() {
-        Account.init(extension.getDatabaseManager(), mock(BlockchainProcessor.class), new BlockchainConfig(), blockchain, null, accountTable, accountGuaranteedBalanceTable,null);
         Transaction deliverTransaction = mock(Transaction.class);
         int height = 1_000_000;
         long txId = 100L;
         long senderId = 200;
-
+        Block lastBlock = mock(Block.class);
```

Comment:
Looks like this snippet is a duplicate of snippet added in previous method

Reply:
I've made the class field

Reply:
moved to class field